services/core-api/Dockerfile.ci

```diff
@@ -0,0 +1,29 @@
+FROM python:3.6
+
+WORKDIR /app
+
+# Update installation utility
+RUN apt-get update
+
+# Install the requirements
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+# Fix log path permissions
+# This is a bit hacky but other fixes require changing app code
+RUN mkdir -p /var/log/core-api
+RUN chmod -R 700 /var/log
+RUN chown -R 1000:0 /var/log
+
+# Setup user to represent developer permissions in container
```

Comment:
Do we need to represent developer permissions in remote environments?

Reply:
It appears that the user is set up with the same permissions as local dev, and is also used to run the API server.

Unless there's some sort of security / permissions problem that I don't see here, I don't see this as an issue. If having the generated 'python' user run the service with similar permissions to a dev makes it work and work securely, this is fine to leave as is.